cluster/network.go

```diff
@@ -323,7 +339,7 @@ var EtcdClientPortList = []string{
 }
 
 var CalicoNetworkLabels = []string{CalicoNodeLabel, CalicoControllerLabel}
-var IPv6CompatibleNetworkPlugins = []string{CalicoNetworkPlugin}
+var IPv6CompatibleNetworkPlugins = []string{CalicoNetworkPlugin, AciNetworkPlugin}
```

Comment:
@LavanyaVaddavalli I think we need to scope the ipv6 compatibility only to `ACI >= 5.2.7.1` (or kubernetes versions that support this plugin version). If users update CIDR to comma separated values for ipv6 but don't upgrade to the new Kubernetes versions, this will break the ACI deploy because the older template renders it directly (same for all fields below) 
` "node-subnet": "{{.NodeSubnet}}",` 
vs 
`"node-subnet": [{{- range $index, $item := .NodeSubnet }}{{- if $index}},{{end}}{{$item}}{{end}}]`. 

Is this a valid scenario or something that's obvious to users by default? I am okay keeping it as is if you think users would be aware of which Kubernetes versions to run before trying to use ipv6.

Reply:
@kinarashah We mention to the user which kubernetes versions are supported for a particular aci release. But if the user still missed updating the kubernetes version, then this is a valid scenario. 

Since we do not take the ACI version as input anywhere, we would have to put a validation check based on kubernetes version { >=1.23.16-rancher2-3 <1.24.0-rancher0, >=1.24.13-rancher2-2 <1.25.0-rancher0, >=1.25.9-rancher2-2) The kubernetes version should satisfy any one of the conditions to be eligible. Is there any pointers on how to add this check for kubernetes versions?

Reply:
Addressed in 4471ba579916db076fbace46d2e89a3d4277e7fa

Reply:
Thanks for updating, confirmed that the range restrictions for ipv6 look right to not break templates now 
```
">=1.23.16-rancher2-3 <1.24.0-rancher0":    aciv5271
">=1.24.13-rancher2-2":                     aciv5271
```